File: Java/UberVan.java
package Java;
import java.util.Map;
import java.util.ArrayList;

public class UberVan extends Car{

    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
  
    public UberVan(String licence, Account driver){
        super(licence, driver);

    }

    @Override
    public void setPassenger(Integer passenger) {

        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typerCarAccepted = []
    seatsMaterial    = []

    def __init__(self, licence, driver, typeCarAccepted, seatsMaterial):
        super().__init__(licence, driver)
        self.typerCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial